package.json

```diff
@@ -157,10 +157,16 @@
             }
           }
         },
+        "quarkus.tools.formatting.insertSpaces": {
```

Comment:
Should not be `quarkus.tools.formatting.insertSpacesBetweenAssign` ?

Reply:
*.surroundEqualsWithSpaces ?

Reply:
fixed

Reply:
@xorye it seems @fbricon would prefer `quarkus.tools.formatting.surroundEqualsWithSpaces`? If it that please do the fix and do the same thing for Quarkus LS settings.

Reply:
fixed with @fbricon 's suggestion